setup.py

```diff
@@ -89,7 +94,7 @@ def run_tests(self):
     python_requires=">=3.8",
     install_requires=install_requires,
     license="MIT",
-    tests_require=tests_require,
+    extras_require=tests_require,
```

Comment:
Is there value in dropping `tests_require` rather than using both? How about:

```
[..]
extras_require=extras_require,
tests_require=extras_require['tests'],
[..]
```